src/main/java/io/leonis/subra/game/data/Referee.java

```diff
@@ -98,6 +109,7 @@
     private final Command command;
     private final Set<Team> teams;
     private final double commandTimeStamp;
+    private final TeamIdentity positiveHalfTeam, negativeHalfTeam;
```

Comment:
if we compress these, then what about compressing `commandTimeStamp` and `commandCount`. also is `commandCount` a double (not an int)?